lesson_01/Angelica Castillo HTML/readme.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="deco.css">
+</head>
+<body>
+   <h2>Angelica Castillo</h2>
+    <ul>
+        <li>Angelica is the daughter of immigrant parents. She is 1 of 6 children. She has worked in retail and has expirience working in Early Childhood Education. Taught at Wilmington Head Start, Bush Early Education, & Mount Pleasant Elementary. Angelica is persuing a software development carrear at Code Differently.</li>
```

Comment:
Styling is still a bit off from the REAME. Background color should be black or white, font should be consistent for the text throughout (font is bigger here than in other sections for some reason). Also, the title (your name) should be left aligned with a line underneath it.